environments.yml

```diff
@@ -0,0 +1,6 @@
+name: planutils
+dependencies:
+  - python=3.9
+  - pip
+  - pip:
+      - -e .
```

Comment:
What do these do?

Reply:
This commit was intended to make the system easily testable/debuggable in an environment where anaconda is already installed.
I initially thought #35 was referring to moving away from Vagrant to anaconda that you are using to isolate the development environment.
However, reading #35 carefully, what you seem to have in mind is to download each planner as a conda package, rather than manually downloading it.
So that is the meaning of my comment "this looks a bit different".

Now "-e ." translates to "pip -e ." i.e. installing the package in the "editable mode".
The source code is not copied to the `site-packages` directory, instead it is linked from there to the current respositiory so that the source code update will be reflected when you launch python.

Reply:
https://pip.pypa.io/en/stable/cli/pip_install/